### PR TITLE
Use @puppeteer/browsers for launching

### PR DESCRIPTION
**Pull request by OrKoN**
Bug: #535

### CODE REVIEW

File: runBiDiServer.sh
Comment (thiagowfx) [inline runBiDiServer.sh:22]: Can we keep using `dev` here?

File: src/bidiServer/index.ts
Comment (thiagowfx) [inline src/bidiServer/index.ts:94]: Double-check: Is empty string included here? It doesn't seem like so: http://shortn/_264g710TVY

File: src/bidiServer/index.ts
Comment (thiagowfx) [inline src/bidiServer/index.ts:132]: Update this comment

File: src/bidiServer/index.ts
Comment (thiagowfx) [inline src/bidiServer/index.ts:138]: Can we do better than `any` here? At the very least, `as string`. But we should have the correct type from puppeteer's `.d.ts` file.

File: src/bidiServer/index.ts
Comment (thiagowfx) [inline src/bidiServer/index.ts:126]:
```suggestion
    // keep-sorted start
    '--allow-pre-commit-input',
    '--disable-background-networking',
    '--disable-background-timer-throttling',
    '--disable-backgrounding-occluded-windows',
    '--disable-breakpad',
    '--disable-client-side-phishing-detection',
    '--disable-component-extensions-with-background-pages',
    '--disable-component-update',
    '--disable-default-apps',
    '--disable-dev-shm-usage',
    '--disable-extensions',
    // AcceptCHFrame disabled because of crbug.com/1348106.
    '--disable-features=Translate,BackForwardCache,AcceptCHFrame,MediaRouter,OptimizationHints',
    '--disable-hang-monitor',
    '--disable-ipc-flooding-protection',
    '--disable-popup-blocking',
    '--disable-prompt-on-repost',
    '--disable-renderer-backgrounding',
    '--disable-sync',
    '--enable-automation',
    '--enable-features=NetworkServiceInProcess2',
    '--export-tagged-pdf',
    '--force-color-profile=srgb',
    '--metrics-recording-only',
    '--no-first-run',
    '--password-store=basic',
    '--use-mock-keychain',
    '--remote-debugging-port=9222',
    // keep-sorted end
```

This will be effective once https://github.com/GoogleChromeLabs/chromium-bidi/pull/539 is merged

**Review by thiagowfx — APPROVED** [review]

**Comment by OrKoN** [comment]
@thiagowfx just to clarify this is not ready for review, the PR is a draft.

File: src/bidiServer/index.ts
Comment (OrKoN) [reply src/bidiServer/index.ts]: ChromeReleaseChannel | undefined are allow here.

File: src/bidiServer/index.ts
Comment (OrKoN) [reply src/bidiServer/index.ts]: Let's add it once the other PR lands.

**Comment by OrKoN** [comment]
@sadym-chromium it looks like headfull WPT might fail. Do you know what is different there?

**Review by sadym-chromium — APPROVED** [review]
LGTM if the tests are passing

**Comment by OrKoN** [comment]
The failure is the same as on main: https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4425152927/jobs/7759850524 related to the screenshot height in WPT

**Comment by thiagowfx** [comment]
LGTM

**Pull request merged**
